media-source-respec.html

```diff
@@ -1160,15 +1223,67 @@ <h2><dfn>MediaSourceHandle</dfn> Object</h2>
+          reality. For example, extensions or internal implementations of postMessage using broadcast may cause
+          unintended multiple recipients of a transferred {{MediaSourceHandle}}. For this reason, implementations are
+          guided to not resolve which potential clone of a transferred {{MediaSourceHandle}} is still valid for
+          attachment until and unless any handle for the underlying {{MediaSource}} object is used in the asynchronous
+          portion of the media element's resource selection algorithm. This is similar to the existing behavior for
+          attachment via [=MediaSource object URLs=], which can be cloned easily, where such a URL is valid for at most
+          one attachment start (across all of its potentially many clones).
+        </p>
+
+        <p>
+          Implementations MUST support at most one attachment (load) via {{HTMLMediaElement/srcObject}} ever for the
+          {{MediaSource}} object underlying a {{MediaSourceHandle}}, regardless of potential cloning of the
+          {{MediaSourceHandle}} due to varying implementations of {{Transferable}}.
+          <p class="note">See [=attaching to a media element=] for how this is enforced during the asynchronous portion
+            of the media element's resource selection algorithm.</p>
+        </p>
+
+        <p>{{MediaSourceHandle}} is only exposed on {{Window}} and {{DedicatedWorkerGlobalScope}} contexts, and cannot
+          successfully transfer between different agent clusters. Transfer of a {{MediaSourceHandle}} object can only
+          succeed within the same agent cluster.
+          <p class="note">For example, transfer of a {{MediaSourceHandle}} object from either a {{Window}} or
+          {{DedicatedWorkerGlobalScope}} to either a SharedWorker or a ServiceWorker will not succeed. Developers should
+          be aware of this difference versus [=MediaSource object URLs=] which are {{DOMString}}s that can be
+          communicated many ways. Even so, [=attaching to a media element=] using a [=MediaSource object URL=] can only
+          succeed for a {{MediaSource}} that was constructed in a {{Window}} context.</p>
+        </p>
+
+        <p>[=Transfer steps=] for a {{MediaSourceHandle}} object MUST include the following steps:
+          <ol>
+            <li>If the {{MediaSourceHandle}}'s {{MediaSourceHandle/[[Detached]]}} internal slot is true, then the
+              [=transfer steps=] must fail by throwing a {{DataCloneError}} exception.</li>
+            <li>If the {{MediaSourceHandle}}'s {{MediaSourceHandle/[[has ever been assigned as srcobject]]}} internal
+              slot is true, then the [=transfer steps=] must fail by throwing a {{DataCloneError}} exception.</li>
+            <li>Otherwise, the {{MediaSourceHandle/[[Detached]]}} internal slot must be set true as part of execution of
+              the [=transfer steps=].</p>
```

Comment:
Similarly performed by the caller.

Reply:
Ack. Removed this redundant step in 7aaebb3